lib/rubocop/cop/rspec/named_subject.rb

```diff
@@ -59,6 +59,11 @@ class NamedSubject < Cop
         def_node_search :subject_usage, '$(send nil? :subject)'
```

Comment:
Do you think [this](https://github.com/rubocop-hq/rubocop-rspec/blob/master/lib/rubocop/rspec/language.rb#L116) existing helper may come handy (to also detect `subject!`)?